apps/services/user-profile/src/app/v2/test/me-user-profile.spec.ts

```diff
@@ -331,7 +331,7 @@ describe('MeUserProfile', () => {
       expect(res.status).toEqual(400)
       expect(res.body).toMatchObject({
         statusCode: 400,
-        message: 'Email verification with hash 000 does not exist',
+        message: 'Email code is not a match. 3 tries remaining.',
```

Comment:
Weren't we going to allow maximum of 3 tries, so 2 tries should be remaining?
We should add a test case to test that we return the error about max tries reached.

Reply:
fixed with comment above where I should set `verification = await verification.increment('tries')`
And I will add test :D